helpdesk/views/staff.py

```diff
@@ -457,7 +457,7 @@ def update_ticket(request, ticket_id, public=False):
     if request.FILES:
         import mimetypes, os
         for file in request.FILES.getlist('attachment'):
-            filename = file.name.encode('ascii', 'ignore')
+            filename = file.name
```

Comment:
Are you sure this change does not break things in python 2.7 when using non-ascii characters in filenames?
The ``'.encode('ascii', ignore')`` effectively removes non-ascii characters from unicode.